functions_build.sh

```diff
@@ -197,9 +197,13 @@ cleanup_install_dir() {
                         ;;
                     *)
                         execute strip lib/lib*.a
-                        # On CentOS 5, libffi and OpenSSL install to lib64/.
+                        # On CentOS 5, libffi and OpenSSL install to lib64/
+                        # by default. To have all libs under lib/, required
+                        # files are copied by chevahbs scripts during build.
```

Comment:
"chevahbs" sounds like a cross between "cevapcici" and "kebabs" :smile: 

I thought it was a typo, then I found it was "Chevah Build Script".